doc/user/content/guides/postgres-cloud.md

```diff
@@ -123,6 +121,24 @@ As a user with the `cloudsqlsuperuser` role, make these changes to the upstream
 
 For more information, see the [Cloud SQL](https://cloud.google.com/sql/docs/postgres/replication/configure-logical-replication#configuring-your-postgresql-instance) documentation.
 
+## DigitalOcean Managed Postgres
+
+Connect to the DigitalOcean Managed Postgres with the `doadmin` user and make the following:
+
+1. The Materialize replica will need access to connect to the upstream database. In your DigitalOcean Managed Postgres console, add your [Materialize instance IP address](/docs/cloud/security/#static-ip-addresses) to the [Trusted Source list](https://docs.digitalocean.com/products/databases/postgresql/how-to/secure/#firewalls).
+
+1. Create a [publication](https://www.postgresql.org/docs/current/logical-replication-publication.html) with the tables you want to replicate:
+
+    ```sql
+    CREATE PUBLICATION mz_source FOR TABLE table1, table2;
+    ```
+
+     As the `doadmin` user, is not a superuser, you will not be able to create a publication for all tables.
+
+     The `mz_source` publication will contain the set of change events generated from the specified tables, and will later be used to ingest the replication stream.
```

Comment:
Don't need an ordered list since the first list item is just intro, how about:

The Materialize replica will need firewall access to connect to the upstream database. In the DigitalOcean console, add your [Materialize instance IP address](/docs/cloud/security/#static-ip-addresses) to the [Trusted Source list](https://docs.digitalocean.com/products/databases/postgresql/how-to/secure/#firewalls) for your Managed PostgreSQL Cluster.

Connect to your PostgreSQL cluster as the `doadmin` user and create a [publication](https://www.postgresql.org/docs/current/logical-replication-publication.html) with the tables you want to replicate:

  ```sql
  CREATE PUBLICATION mz_source FOR TABLE table1, table2;
  ```
**Note:** Because the `doadmin` user, is not a superuser, you will not be able to create a publication for **all** tables.

The `mz_source` publication will contain the set of change events generated from the specified tables, and will later be used to ingest the replication stream.

Reply:
Good point! Just made that change!